src/centralserver/admin-ui/src/views/InitialConfiguration/InitialConfiguration.vue

```diff
@@ -292,50 +292,38 @@ export default (
       if (!this.disabledFields.pin) {
         formData.software_token_pin = this.pin;
       }
-      await this.resetNotifications();
+      this.resetNotifications();
       await this.initalizationRequest(formData)
-        .then(
-          () => {
-            this.$router
-              .push({
-                name: RouteName.Members,
-              })
-              .catch(swallowRedirectedNavigationError);
-          },
-          (error: AxiosError) => {
-            const errorInfo: ErrorInfo = error.response?.data || { status: 0 };
-            if (isFieldError(errorInfo)) {
-              const fieldErrors = errorInfo.error?.validation_errors;
-              if (fieldErrors) {
-                const identifierErrors: string[] = getTranslatedFieldErrors(
-                  'initialServerConf.instanceIdentifier',
-                  fieldErrors,
-                );
-                const addressErrors: string[] = getTranslatedFieldErrors(
-                  'initialServerConf.centralServerAddress',
-                  fieldErrors,
-                );
-                this.$refs.initializationForm.setErrors({
-                  'init.identifier': identifierErrors,
-                  'init.address': addressErrors,
-                });
-                this.showError(error);
-              }
-              return;
-            }
-            if (invalidParamsErrors(errorInfo).length > 0) {
```

Comment:
This simplification was a good finding!
Is there any use left for the functions `invalidParamsErrors` and `isWeakPinError`? If not the function definitions could be removed also...

Reply:
Seems that they are not used. I removed those now.